certificate-service/clinicalprocess-certificate-v4/src/main/java/se/inera/intyg/certificateservice/infrastructure/clinicalprocesscertificatev4/certificate/XmlGeneratorValue.java

```diff
@@ -77,7 +86,7 @@ private List<XmlMapping> mappings(CertificateModel certificateModel,
   }
 
   private static boolean noCustomMapping(XmlMapping mapping) {
-    return mapping.getMapping() == null;
+    return mapping.getMapping() == null || mapping.getMapping().elementValueMapper().isPresent();
```

Comment:
I think it would be good if we changed the logic here a little bit. As it is written now we can't have custom mapping if we provide a elementValueMapper. But let's say we would have liked to have a custom mapper and a an elementValueMapperr?

Maybe use something like this instead: 
`return mapping.getMapping() == null || mapping.getMapping().elementId() == null;`